modules/files/files_test.go

```diff
@@ -74,6 +74,19 @@ func TestCopyFolderContentsWithSymLinks(t *testing.T) {
 func TestCopyFolderContentsWithBrokenSymLinks(t *testing.T) {
 	t.Parallel()
 
+	// Creating broken symlink
+	fPath := filepath.Join(copyFolderContentsFixtureRoot, "symlinks-broken/nonexistent-folder/bar.txt")
+	sPath := filepath.Join(copyFolderContentsFixtureRoot, "symlinks-broken/bar.txt")
```

Comment:
what do the `s` and `f` stand for in these names? `soft` and `file`?

Reply:
file and symlink. Could've named it `filePath`, instead of `fPath`, but thought it could be confusing given the library being used is `filepath`.

Reply:
Perhaps `pathToFile` and `pathToSymlink`?